docs/tutorial.rst

```diff
@@ -110,37 +111,40 @@ Let's try our luck with a bivariate normal distribution::
 
     fit = Fit(model, x=xdata, y=ydata, p=pdata)
 
-By using the magic of named models, the flow of information is still very clear, even with such a complicated function.
+By using the magic of named models, the flow of information is still relatively clear, even with such a complicated function.
 
 This syntax also supports vector valued functions::
 
     model = {y_1: a * x**2, y_2: 2 * x * b}
 
 One thing to note about such models is that now ``model(x=xdata)`` obviously no longer works as ``type(model) == dict``.
-There is a preferred way to resolve this. If any kind of fitting object has been initiated, it will have a ``.model`` atribute
-containing an instance of ``Model``. This can again be called::
+There is a preferred way to resolve this. If any kind of fitting object has been initiated, it will have a :attr:`~symfit.core.fit.Fit.model` atribute
+containing an instance of :class:`~symfit.core.fit.Model`. This can again be called::
 
+    a, b = parameters('a, b')
+    y_1, y_2, x = variables('y_1, y_2, x')
+    
     model = {y_1: a * x**2, y_2: 2 * x * b}
-    fit = Fit(model, x=xdata)
+    fit = Fit(model, x=xdata, y_1=y_data1, y_2=y_data2)
     fit_result = fit.execute()
 
-    y_1, y_2 = fit.model(x=xdata, **fit_result.params)
+    y_1_result, y_2_result = fit.model(x=xdata, **fit_result.params)
 
-This returns a tuple with the components evaluated so through the magic of tuple unpacking``y_1`` and ``y_2`` contain the
-evaluated fit. Nice!
+This returns a :class:`namedtuple` with the components evaluated so through the magic of tuple unpacking ``y_1`` and ``y_2`` contain the
+evaluated fit. The variables will be ordered alphabetically. Nice! It may sometimes be clearer to do the unpacking explicitly though.
 
-If for some reason no ``Fit`` is initiated you can make a Model object yourself::
+If for some reason no :class:`~symfit.core.fit.Fit` is initiated you can make a :class:`~symfit.core.fit.Model` object yourself::
 
-    from symfit import Model
+    model = Model(model_dict)
 
-    model_dict = {y_1: a * x**2, y_2: 2 * x * b}
-    model = Model.from_dict(model_dict)
+    outcome = model(x=xdata, a=2.4, b=0.1)
+    y_1_result = outcome.y_1
+    y_2_result = outcome.y_2
 
-    y_1, y_2 = fit.model(x=xdata, a=2.4, b=0.1)
 
 symfit exposes sympy.api
```

Comment:
sympy.api -> sympy since sympy.api does not actually exist. Mistake from working with a lot of Enthought projects that did follow that paradigm at the time.